source/reference/minio-mc/mc-batch-describe.rst

```diff
@@ -0,0 +1,103 @@
+.. _minio-mc-batch-describe:
+
+=====================
+``mc batch describe``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc batch describe
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+Syntax
+------
+
+.. start-mc-batch-describe-desc
+
+The :mc:`mc batch describe` command outputs the job definition for a specified job ID.
+
+.. end-mc-batch-describe-desc
+
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command outputs a list of all jobs currently in progress on the ``myminio`` alias.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc batch describe myminio KwSysDpxcBU9FNhGkn2dCf
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] batch describe TARGET           \
+                                         JOBID
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: TARGET
+   :required:
+   
+   The :ref:`alias <alias>` and bucket on the MinIO deployment to which to add the object lifecycle management rule. 
+
+.. mc-cmd:: JOBID
+   :optional:
+
+   The unique identifier of a job to summarize.
+   To find the ID of a job, use :mc:`mc batch list`.
```

Comment:
If omitted this seems to do listing-type behavior.

Reply:
More of that copypasta fail.